plugins/communications-app/ScheduleSection/index.jsx

```diff
@@ -99,13 +103,15 @@ const ScheduleSection = ({ openTaskAlert }) => {
     complete: intl.formatMessage(messages.ScheduleSectionSubmitButtonComplete),
     completeSchedule: intl.formatMessage(messages.ScheduleSectionSubmitButtonCompleteSchedule),
     error: intl.formatMessage(messages.ScheduleSectionSubmitButtonError),
+    loadingTeams: intl.formatMessage(messages.ScheduleSectionSubmitButtonDefault),
   }), [intl]);
 
   const statefulButtonDisableStates = useMemo(() => [
     'pending',
     'complete',
     'completeSchedule',
-  ], []);
+    isLoadingTeams ? 'loadingTeams' : '',
```

Comment:
Same about this states:

```suggestion

    COMPLETE_SCHEDULE,
    isLoadingTeams ? LOADING_TEAMS : '',
```